pkgs/development/libraries/libjwt/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, autoreconfHook, pkg-config, jansson, openssl }:
```

Comment:
```suggestion
{ stdenv, lib, fetchFromGitHub, autoreconfHook, pkg-config, jansson, openssl }:
```

Reply:
`fetchFromGitHub` is a better way to fetch than `fetchurl`
Source: https://nixos.org/manual/nixpkgs/stable/#sec-sources